main.go

```diff
@@ -25,6 +23,18 @@ func main() {
 	app.Version = VERSION
 	app.Usage = "An external resource based scheduler for Rancher."
 	app.Action = run
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "metadata-address",
+			Usage: "The metadata service address",
+			Value: "rancher-metadata",
+		},
+		cli.StringFlag{
+			Name:  "listen",
```

Comment:
I noticed we used the term `listen` elsewhere, so I followed it. `port` would be more descriptive.